tests/const.py

```diff
@@ -114,6 +114,9 @@
         "post_nl_delivered",
         "post_nl_delivering",
         "post_nl_packages",
+        "ebay_delivering",
```

Comment:
```suggestion
```
Since eBay doesn't actually deliver packages we wouldn't add a "delivering" sensor.